django_cte/raw.py

```diff
@@ -19,15 +19,17 @@ def get_source_expressions(self):
             return []
 
     class raw_cte_compiler(object):
-        @staticmethod
-        def as_sql():
+        def as_sql(self, ):
             return sql, params
 
+        def quote_name_unless_alias(self, name):
+            return name
```

Comment:
Should this use `qn(name)`? Could also make this a `@staticmethod` and undo the change on line 32 below since `self` is not referenced.

Reply:
It actually is the value of qn used in CTECompiler.generate_sql if the query is a raw query.
for non raw queries this function comes from the base django.db.models.sql.compiler.SQLCompiler which use the connection to provide the correct escaping function. I was just ignoring escaping raw queries. 

I have now added a test to check that raw queries to escape the CTE name.
A static would have worked, but fixing escaping the raw cte name requires it again as we need to store the connection.